### PR TITLE
Widgets updated instantly.

### PR DESCRIPTION
**Pull request by free4murad**
Solves issue #192 
@yatna @chhavip 
Please review the PR.

### CODE REVIEW

**Comment by free4murad** [comment]
@yatna 
i will set the update time to 1 day.

**Comment by free4murad** [comment]
@yatna 
Done please review.

File: malaria-app-android/src/main/res/xml/appwidget_info.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <appwidget-provider xmlns:android="http://schemas.android.com/apk/res/android"
     android:minWidth="200dp"
     android:minHeight="40dp"
-    android:updatePeriodMillis="100"
+    android:updatePeriodMillis="86400000"
```
Comment (chhavip) [inline malaria-app-android/src/main/res/xml/appwidget_info.xml]: Can we not hardcode this and instead put in integer.xml? with an appropriate desc

Comment (free4murad) [inline malaria-app-android/src/main/res/xml/appwidget_info.xml]: @chhavip 
there is no integer.xml file.
Should I create one??

**Comment by free4murad** [comment]
@chhavip 
done, please review.

**Comment by chhavip** [comment]
@yatna, looks good to me now, if approved, will merge.

**Comment by free4murad** [comment]
@yatna @chhavip 
I have changed the base branch to `develop`.
Also I have removed multiple commits.
Please review.

**Comment by free4murad** [comment]
@yatna 
please review this

**Comment by chhavip** [comment]
Looks good, merging.

**Pull request merged**